clang/test/SemaHLSL/Semantics/position.ps.hlsl

```diff
@@ -2,6 +2,6 @@
 // RUN: %clang_cc1 -triple spirv-pc-vulkan1.3-pixel -finclude-default-header -x hlsl -verify -o - %s
 
 float4 main(float4 a : A) : SV_Position {
-// expected-error@-1 {{attribute 'SV_Position' is unsupported in 'pixel' shaders, requires one of the following: pixel, vertex}}
+// expected-error@-1 {{semantic 'SV_Position' is unsupported as output for stage pixel}}
```

Comment:
This message seem less informative, and does not read as well.

Could you do something like:

semantic 'SV_Position' is unsupported in 'pixel' shaders as an output, requires one of the following: pixel input, vertex input or output